plugin/src/test/groovy/org/gradle/testretry/testframework/SpockFuncTest.groovy

```diff
@@ -853,7 +840,7 @@ class SpockFuncTest extends AbstractFrameworkFuncTest {
         return """
             dependencies {
                 implementation "org.codehaus.groovy:groovy-all:2.5.8"
-                testImplementation "org.spockframework:spock-core:1.3-groovy-2.5"
+                implementation "org.spockframework:spock-core:1.3-groovy-2.5"
```

Comment:
❓ Why change it from `testImplementation` to `implementation`?

Reply:
becuse of https://github.com/gradle/test-retry-gradle-plugin/pull/103#discussion_r642856587, but I'll change it to `/test/` instead, I doubt it was deliberate